ttbb/train.py

```diff
@@ -316,14 +346,14 @@ def on_batch_end(self, batch, logs={}):
 #################################
 #Keras model compile and training
 #################################
-nvar = len(ut.getVarlist())
-a = 300
+nvar = data.shape[1]
+a = 50
 b = 0.08
 init = 'glorot_uniform'
 
 with tf.device("/cpu:0") :
     inputs = Input(shape=(nvar,))
-    x = Dense(a, kernel_regularizer=l2(1E-2))(inputs)
+    x = Dense(5, kernel_regularizer=l2(1E-2))(inputs)
```

Comment:
Aren't we supposed to use "a" instead of hard-coded  value "5"?

Reply:
You are right. It's a residue after the testing process. I will fix this.